change/@itwin-geo-tools-react-7351df52-2f79-431a-8e06-0ab13c712ca6.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Update geo-tools-react to appui 5.2.0 and itwinui 3.16.5.  Replace deprecated components with itwinui components. Fix broken geo tools search address ui.",
```

Comment:
you also ended up bumping core too

Reply:
Oh right - I will add that to the PR description

Reply:
Rolled back to 4.0.0

Reply:
```suggestion
  "comment": "Add support for AppUI@5.x and React@18.x. Switch to using iTwinUI components. Fix broken geo tools search address UI.",
```